vars/kola.groovy

```diff
@@ -7,8 +7,8 @@
 //    arch:               string   -- the target architecture
 //    cosaDir:            string   -- cosa working directory
 //    parallel:           integer  -- number of tests to run in parallel (default: # CPUs)
-//    skipBasicScenarios  boolean  -- skip basic qemu scenarios
-//    skipSecureBoot      boolean  -- skip secureboot tests
+//    skipBasicScenarios  boolean  -- skip basic qemu scenarios, as of cosa v0.17.0 these tests are part of kola. consquently this option is deprecated
+//    skipSecureBoot      boolean  -- skip secureboot tests, as of cosa v0.17.0 these tests are part of kola. consquently this option is deprecated
```

Comment:
We should keep this option, so it's not deprecated. See https://github.com/coreos/coreos-ci-lib/pull/154#discussion_r1388624054.

Reply:
oook, yes can do.